service/src/main/kotlin/fi/tampere/trevaka/emailclient/config/EmailConfiguration.kt

```diff
@@ -59,11 +59,11 @@ internal class EmailMessageProvider(): IEmailMessageProvider {
             Varhaiskasvatus ja esiopetus
             Asiakaspalvelu
             <a href="mailto:varhaiskasvatus.asiakaspalvelu@tampere.fi">varhaiskasvatus.asiakaspalvelu@tampere.fi</a>
-            <a href="tel:+358408007260">040&nbsp;800&nbsp;7260</a>, ma-pe klo 9-12
+            Puh. <a href="tel:+358408007260">040&nbsp;800&nbsp;7260</a> (ma-pe klo 9-12)

```

Comment:
BTW, should there be an ndash or hyphen in between ma and pe and times?

Reply:
See #89